web/client/themes/default/less/geostory.less

```diff
@@ -993,8 +993,12 @@
                 .ms-column-contents {
                     .ms-content {
                         padding: 0 15%;
+                        &:not(.ms-content-text) {
+                            padding-bottom: 16px;
+                            padding-top: 16px;
+                        }
```

Comment:
So the padding exists for every content that is not textual? why?

Reply:
Under [ms-content-text](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/themes/default/less/geostory.less#L1270), [padding ](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/themes/default/less/geostory.less#L1393) already exists, so I skipped adding more padding.